server/src/weaverbird/pipeline/steps/replacetext.py

```diff
@@ -2,12 +2,15 @@
 
 from pydantic import Field
 
-from weaverbird.pipeline.steps.utils.base import BaseStep
+from weaverbird.pipeline.steps.utils.base import BaseStep, to_camelcase
 from weaverbird.pipeline.steps.utils.render_variables import StepWithVariablesMixin
 from weaverbird.pipeline.types import ColumnName, TemplatedVariable
 
 
 class ReplaceTextStep(BaseStep):
+    class Config:
+        alias_generator = to_camelcase
```

Comment:
isn't it something we should have by default on all base steps ?

Reply:
I agree, but It'll probably be a breaking change. I'm preparing another PR for that, to be merged after the LTS branches have been created

Reply:
#1559